src/js/mixins/DuplicableRowsMixin.jsx

```diff
@@ -40,7 +40,7 @@ var DuplicableRowsMixin = {
     Object.keys(duplicableRowsScheme).forEach(function (fieldId) {
       if (state.rows[fieldId] == null || state.rows[fieldId].length === 0) {
         let rowScheme = Util.extendObject(duplicableRowsScheme[fieldId], {
-          consecutiveKey: Util.getUniqueId()
+          consecutiveKey: parseInt((Util.getUniqueId() + "").slice(-9))
```

Comment:
Taking this kinda small risk is fine for me, but I wonder what's causing these incredible high indices, you're trying to workaround, in the first place? Shouldn't a simple consecutive number also do the trick?